contracts/blade/Gateway.sol

```diff
@@ -92,6 +92,10 @@ contract Gateway is ValidatorSetStorage, IGateway {
 
         verifySignature(bls.hashToPoint(DOMAIN_BRIDGE, hash), signedBridgeBatch.signature, signedBridgeBatch.bitmap);
 
+        if (block.number > signedBridgeBatch.threshold && !signedBridgeBatch.isRollback) {
+            revert("block number is bigger than threshold");
```

Comment:
Consider changing the revert message to something like this, e.g.
`batch [start id-end id] has timed out`.